docker/bridge.dockerfile

```diff
@@ -1,3 +1,3 @@
 FROM golang:1.18
 
-RUN go install github.com/litentry/ChainBridge/cmd/chainbridge@dev
+RUN go install github.com/litentry/ChainBridge/cmd/chainbridge@minqi-dev
```

Comment:
Why don't use `dev` branch?

Reply:
dev is phala's smart contract. minqi-dev is our new smart contract.